roles/openshift_prometheus/README.md

```diff
@@ -27,6 +27,7 @@ For default values, see [`defaults/main.yaml`](defaults/main.yaml).
 Each prometheus component (prometheus, alertmanager, alert-buffer, oauth-proxy) can set pv claim by setting corresponding role variable:
 ```
 openshift_prometheus_<COMPONENT>_storage_type: <VALUE>
+openshift_prometheus_<COMPONENT>_storage_class: <VALUE>
 openshift_prometheus_<COMPONENT>_pvc_(name|size|access_modes|pv_selector): <VALUE>
 ```
 e.g
```

Comment:
Could you also add this example here:
openshift_prometheus_storage_class: gluster-dyn